docs/release-notes/highlights-2.7.0.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[release-highlights-2.7.0]]
+== 2.7.0 release highlights
+
+[float]
+[id="{p}-270-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 2.7.0 of {n}. Check <<release-notes-2.7.0>> for the full list of changes.
```

Comment:
Should we have at least one meaning full sentence (or leave out release hightligths altogether). Something along the lines of this is a maintenance release see the release notes for full list of changes.

Reply:
Yes, I highlighted 2 changes in [20944b5](https://github.com/elastic/cloud-on-k8s/pull/6566/commits/20944b504ec84aa5b75a7170364d4234723f71a1).